src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -440,7 +440,7 @@ private void FinishRequest(EasyRequest completedOperation, int messageResult)
                         completedOperation.EnsureResponseMessagePublished();
                         break;
                     default:
-                        completedOperation.FailRequest(CreateHttpRequestException(GetCurlException(messageResult)));
+                        completedOperation.FailRequest(CreateHttpRequestException(new CurlException(messageResult, false)));
```

Comment:
Nit: I personally find it easier to understand Boolean arguments to functions when they're named at the call site, e.g.
```C#
new CurlException(messageResult, isMulti: false);
```